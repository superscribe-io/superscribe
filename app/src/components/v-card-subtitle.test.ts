import { test, expect } from 'vitest';
import { mount } from '@vue/test-utils';

import VCardSubtitle from './v-card-subtitle.vue';

test('Mount component', () => {
	expect(VCardSubtitle).toBeTruthy();

	const wrapper = mount(VCardSubtitle, {
		slots: {
			default: 'Slot Content',
		},
	});

	expect(wrapper.html()).toMatchSnapshot();
});
