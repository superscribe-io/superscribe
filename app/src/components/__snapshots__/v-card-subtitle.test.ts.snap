// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Mount component 1`] = `"<div data-v-f6658148=\\"\\" class=\\"v-card-subtitle\\">Slot Content</div>"`;
